client/beta/beta.css

```diff
@@ -169,13 +169,19 @@
   padding-right: 48px;
 }
 
-.faqFeature {
+.faqFeature, .faqTOC {
   composes: row itemsCenter;
   composes: pageWidth;
   padding: 48px 0;
   border-bottom: 1px solid colorDividers;
 }
 
+.faqTOC {
```

Comment:
same as above, acronyms only get the first letter capitalized (e.g. `Rpc`, `Http`, etc.)